src/containers/Media/Media.js

```diff
@@ -25,7 +25,17 @@ export default class extends Component {
   render() {
     return (
       <div className={style.block}>
-        <img className={style.image} alt={this.props.image} src={this.props.media.selectedImage || this.props.image} />
+        <picture className={style.image}>
+          {/*<!--[if IE 9]><video style="display: none;"><![endif]-->*/}
```

Comment:
We should probably remove `style="display: none;"`. Writing a CSS rule like

```
picture > video {
  display: none;
}
```
is cleaner